ros_speech_recognition/README.md

```diff
@@ -30,6 +30,106 @@ This package uses Python package [SpeechRecognition](https://pypi.python.org/pyp
   print result # => 'Hello, world!'
   ```
   
+If you are using `ros_speech_recognition` with `~continuous` is `True`, you can subscribe `/Tablet/voice` (`speech_recognition_msgs/SpeechRecognitionCandidates`) message.
+
+1. Launch sample launch file.
```

Comment:
How about adding google cloud speech-to-text URL to README.md too?
It would be helpful.
https://cloud.google.com/speech-to-text/docs/reference/rest/v1/speech/recognize#wordinfo